queries.sql

```diff
@@ -99,3 +100,68 @@ SELECT name FROM animals WHERE owner_id = (SELECT id FROM owners WHERE full_name
 
 -- Who owns the most animals?
 SELECT full_name, COUNT(*) FROM owners LEFT JOIN animals ON owners.id = animals.owner_id GROUP BY full_name ORDER BY COUNT(*) DESC LIMIT 1;
+
+-- QUERIES USING JOIN TABLES
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name FROM vets 
+  JOIN visits ON vets.id = visits.vets_id
+  JOIN animals ON animals.id = visits.animals_id
+  WHERE vets.name = 'William Tatcher'
+  ORDER BY visits.date_of_visit DESC
+  LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(*) as num_animals_visited from vets
+	JOIN visits ON vets.id = visits.vets_id
+	WHERE vets.name = 'Stephanie Mendez';
```

Comment:
- [ ] To select _different_ animals kindly use DISTINCT instead of COUNT, similar to:
  ```sql
  SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```

Reply:
Dear @rloterh I appreciate the feedback however I don't think this should be listed as a required change as the [Project Requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#project-requirements) do not specify the method that should be used to get an answer to the questions.

Reply:
@jsug9 
Kindly note that we're actually looking at a more efficient solution here, to the question:
_How many **different** animals did Stephanie Mendez see?_
The best case in this scenario is to use **DISTINCT**
Cheers, and happy coding!

Reply:
<img width="772" alt="Captura de Pantalla 2022-06-01 a la(s) 9 05 42 p  m" src="https://user-images.githubusercontent.com/75434179/171532626-c3761c7c-04a8-4700-b3ca-5b828d596369.png">

https://github.com/jsug9/Vet-Clinic-Database/blob/1acd05c980ed6782daffd7c0710dbdf47e0cb907/queries.sql#L113

Dear @rloterh I have just tried this code and it is not really more efficient, it returns a list of animals, however, the question is asking **How many** different animals did Stephanie Mendez see?. My code return a number which is what the question is asking for. So, I am sorry, but I will not make this change.